### PR TITLE
Added JavaDocs

### PR DESCRIPTION
**Pull request by nschnitzer**
Google Code In task

### CODE REVIEW

File: src/main/java/org/terasology/navgraph/Floor.java [hunk 1]
```diff
@@ -61,20 +83,40 @@ public void merge(Region neighbor) {
         }
     }
 
+    /**
+     * Resets Entrances
```
Comment (jellysnake) [inline src/main/java/org/terasology/navgraph/Floor.java]: Like my above comment, i can tell it'll reset entrances from the name of the method, but what does resetting an entrance involve? Does it reset each entrance to some state? Does it delete them? 

A better description here would be "Removes all entrances from the floor, and sets them to null"

Still short and easy to write, it just tells me more than i would get from the name

File: src/main/java/org/terasology/navgraph/Floor.java [hunk 2]
```diff
@@ -31,25 +31,47 @@
     private Entrance[] entranceMap;
     private List<Entrance> entrances;
 
+    /**
+     * Creates a new Floor object
+     * @param navGraphChunk the navGraphChunk parameter, not null
+     * @param id the id parameter, not null
```
[the remark this answers is not in the document]
Comment (jellysnake) [reply src/main/java/org/terasology/navgraph/Floor.java]: To use this one as an example, a better description would be "The id for the new Floor"

Only marginally longer, tells me what the id does, not what it is. I know it's the parameter called `id` what I don't know it's which is it is. Is it the id of the navGraphChunk? Is it the id of something else to be set in the floor? Is it the id of the floor?

[the remark this answers is not in the document]
Comment (nschnitzer) [reply src/main/java/org/terasology/navgraph/Floor.java]: Alright, I was initially doing that. I just saw something on the wiki that made me do what I did.

**Comment by mjuvekar7** [comment]
This looks mostly good to me. Just a few changes:
- Remove all the "not null"s. It seems unnatural, and I feel that it doesn't need to be there (I mean in theory there could be situations where someone would actually want something to be null).
- Add periods (full stops) at the end of the method descriptions. I know this is kinda trivial, but I feel you might as well do it :)

**Pull request merged**